index.html

```diff
@@ -205,175 +189,129 @@
     implemented in any manner, so long as the end result is equivalent. (In
     particular, the algorithms defined in this specification are intended to be
     easy to follow, and not intended to be performant.)</p>
-    <section id="terminology">
+    <section>
       <h2>Dependencies</h2>
       <dl>
         <dt>DOM</dt>
         <dd>
           <p>The following terms are defined in the DOM specification:
           [[!DOM]]</p>
           <ul>
-            <li>the <dfn id='document-url'><a href=
-            "http://www.w3.org/TR/dom/#concept-document-url">document
-            URL</a></dfn></li>
-            <li><dfn id='document'><a href=
-            "http://www.w3.org/TR/dom/#concept-document">document</a></dfn></li>
-            <li><dfn id='throw-a-name-exception'><a href=
-            "http://www.w3.org/TR/dom/#concept-throw">throw a name
-            exception</a></dfn></li>
+            <li>the <dfn data-cite="DOM#concept-document-url">document
+            URL</dfn></li>
+            <li><dfn data-cite="DOM#concept-document">document</dfn></li>
           </ul>
         </dd>
-        <dt>HTML5</dt>
+        <dt>HTML52</dt>
```

Comment:
@marcoscaceres I meant the WHATWG version has dropped the term "referrer source" :)

@igrigorik make sense! Shall we merge this and make that change in a separated pull?